app/controllers/shelters_controller.rb

```diff
@@ -35,7 +35,7 @@ def destroy
 
     redirect_to "/shelters/"
   end
-  
+
   private
 
   def shelter_params
```

Comment:
I like how we didn't have to add anything to our controller. Just iterating over @shelter.reviews is perfect.